packages/next-auth/src/providers/shopify.js

```diff
@@ -0,0 +1,79 @@
+export default function Shopify(options) {
+  const {
+    shop = "",
+    scope = "",
+    clientId = "",
+    clientSecret = "",
+    apiVersion = "",
+  } = options
```

Comment:
```suggestion
  const { apiVersion = "2022-01" } = options
```

Let's set a default value for the apiVersion. `scope` is misplaced, and the others are required values, so no need to default them anything.